samples/xamarin-forms/LoginApp/LoginApp/ViewModels/SignUpViewModel.cs

```diff
@@ -0,0 +1,87 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Reactive;
+using Acr.UserDialogs;
+using LoginApp.ViewModels.Abstractions;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+using ReactiveUI.Validation.Abstractions;
+using ReactiveUI.Validation.Contexts;
+using ReactiveUI.Validation.Extensions;
+using Splat;
+
+namespace LoginApp.ViewModels
+{
+    public class SignUpViewModel : ViewModelBase, ISupportsValidation
+    {
+        private readonly Func<string, bool> _isDefined = value => !string.IsNullOrEmpty(value);
+        private readonly IUserDialogs _dialogs;
+
+        [Reactive] public string UserName { get; set; }
+
+        [Reactive] public string Password { get; set; }
+
+        [Reactive] public string ConfirmPassword { get; set; }
+
+        [Reactive] public ReactiveCommand<Unit, bool> SignUp { get; private set; }
```

Comment:
Not sure if the command needs to be `[Reactive]` and have a private setter. 
Worth making it `get`-only and removing the attribute. Although not as critical.

Reply:
it doesn't… fixing it!